test/example/iss/README.md

```diff
@@ -0,0 +1,81 @@
+# International Space Station (ISS) library
+
+This library accesses the International Space Station's APIs
+(using [package:http](https://pub.dartlang.org/packages/http))
+to fetch the space station's current location.
+
+The unit tests for this library use package:mockito to generate
+preset values for the space station's location,
+and package:test to create reproducible scenarios to verify the
+expected outcome.
+
+The ISS library, `iss.dart`, consists of two classes:
+
+**IssLocator**
+: Fetches the current GPS position directly under the space station.
+
+**IssSpotter**
+: Performs calculations from the observer's location on earth.
+
+---
+
+The unit test, `iss_dart.test`, mocks the IssLocator class:
+
+```
+@proxy
+class MockIssLocator extends Mock implements IssLocator {}
+```
+The tests check for two scenarios:
+
+**Spherical distance**
+: Given two predefined points on earth, verify the calculated distance
+between them.
+
+```
+  group('Spherical distance', () {
+    test('London - Paris', () {
+      Point<double> london = new Point(51.5073, -0.1277);
+      Point<double> paris = new Point(48.8566, 2.3522);
+      double d = sphericalDistanceKm(london, paris);
+      expect(d, closeTo(343.5, 0.1));
+    });
+
+    test('San Francisco - Mountain View', () {
+      Point<double> sf = new Point(37.783333, -122.416667);
+      Point<double> mtv = new Point(37.389444, -122.081944);
+      double d = sphericalDistanceKm(sf, mtv);
+      expect(d, closeTo(52.8, 0.1));
+    });
+  });
+```
+
+** ISS Spotter**
+: Stubs `IssLocator.currentPosition()` using `when().thenReturn()`.
```

Comment:
I don't think currentPosition should have parens here. Would be confusing when below we see it without parens, as if it were a reference to a method, rather than a getter call.